donghyeun/controllers/postController.js

```diff
@@ -0,0 +1,45 @@
+const { ModifyPost } = require("../models/postDao");
+const postService = require("../services/postService");
+
+const Upload = async (req, res) => {
```

Comment:
- 변수명은 카멜케이스로 지어주세요.
- 함수에 대한 변수명(함수명)은 동사형으로 지어주되 목적(대상)까지 나타낼 수 있게 해주면 좋을 것 같습니다. 전체적으로 함수명은 한 번 더 살펴봐 주세요.
```suggestion
const createPost = async (req, res) => {
```